man/soybean.Rd

```diff
@@ -140,7 +140,7 @@
     \item The \code{sowing_time} input to the
     \code{soybean_development_rate_calculator} module is set to 0 because
     Soybean-BioCro uses the weather data to set the sowing time. In other words,
-    the weather data is truncated so it begins at the sowing date.
+    the weather data is truncated so it begins at the beggining of the simulation.
```

Comment:
"beggining" -> "beginning"

Though I'm not sure if I see why this wording needed to change